data/locale/German.json

```diff
@@ -1,186 +1,189 @@
 
     "Network error": "Netzwerkfehler",
-    "check the network settings and try again": "prüfe die Netzwerkeinstellungen und versuche es nochmal",
-    "SSL error": "SSL Fehler",
-    "check your Wii Us date and time settings": "Prüfe die Datums- und Zeiteinstellungen deiner Wii U",
+    "check the network settings and try again": "Überprüf die Netzwerkeinstellungen und versuch es erneut",
+    "SSL error": "SSL-Fehler",
+    "check your Wii Us date and time settings": "Überprüf die Datums- und Zeiteinstellungen deiner Wii U",
     "Next try in _ seconds.": "Nächster Versuch in _ Sekunden.",
-    "Couldn't resolve hostname": "Konnte Host nicht auflöaen",
-    "Couldn't connect to server": "Konnte nicht zu Server verbinden",
-    "Operation timed out": "Zeit abgelaufen",
-    "The server didn't return any data": "Der Server hat keinerlei Daten zurück geschickt",
-    "I/O error": "I/O Fehler",
+    "Couldn't resolve hostname": "Hostname konnte nicht aufgelöst werden",
+    "Couldn't connect to server": "Verbindung zum Server konnte nicht hergestellt werden",
+    "Operation timed out": "Zeitüberschreitung",
+    "The server didn't return any data": "Der Server hat keine Daten zurückgegeben",
+    "I/O error": "I/O-Fehler",
     "Verification failed": "Verifikation fehlgeschlagen",
-    "Handshake failed": "Handschlag fehlgeschlagen",
+    "Handshake failed": "Handshake fehlgeschlagen",
     "Internal error": "Interner Fehler",
-    "Unknown libcurl error": "Unbekannter libcurl Fehler",
-    "Do you really want to cancel?": "Willst du wirklich abbrechen?",
-    "The download of title.tmd failed with error: 404": "Der Download der title.tmd ist mit Fehlercode 404 fehlgelschlagen",
-    "The title cannot be found on the NUS, maybe the provided title ID doesn't exists or\nthe TMD was deleted": "Der Titel kann nicht auf dem NUS gefunden werden. Vielleicht stimmt die Titel ID nicht oder die\nTMD wurde gelöscht",
-    "The download returned a result different to 200 (OK)": "Der Download hat etwas anderes als 200 (OK) zurück gegeben",
-    "Request failed. Try again": "Anfrage gescheitet. Versuche es erneut",
-
-    "Press \uE003 to retry": "Drücke \uE003 um es nochmal zu versuchen",
-    "This is a reliable way to brick your console!\nAre you sure you want to do that?": "Dies ist ein sicherer Weg deine Konsole zu beicken!\nBist du sicher das du dies möchtest?",
-    "Are you really sure you want to brick your Wii U?": "Bist du sicher das du deine Konsole zerstören willst?",
-    "You're on your own doing this,\ndo you understand the consequences?": "Du bist selbst verantwortlich wenn du fortfährst,\nverstehst du die Konsequenzen?",
-
-    "Search": "Suche",
-    "unknown": "unbekannten",
+    "Unknown libcurl error": "Unbekannter libcurl-Fehler",
+    "Do you really want to cancel?": "Möchtest du wirklich abbrechen?",
+    "The download of title.tmd failed with error: 404": "Das Herunterladen von title.tmd schlug mit dem Code 404 fehl",
```

Comment:
The `der title.tmd` was short for `der title.tmd Datei`, so `von title.tmd` sounds wrong to me. Care to elaborate on this change?